src/open_inwoner/accounts/forms.py

```diff
@@ -279,6 +279,17 @@ def send_mail(
         email_message.send()
 
 
+class CategoriesForm(forms.ModelForm):
+    selected_categories = forms.ModelMultipleChoiceField(
+        queryset=Category.objects.published(),
+        widget=forms.widgets.CheckboxSelectMultiple,
+    )
```

Comment:
@alextreme I did notice that this form currently doesn't allow you to uncheck all categories, is that an issue?

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/e7e07f9d-b08d-4aa7-9b6c-86fe21d9f979)

(also the error doesn't seem in line with the design :grimacing:)